6.1.flatlandia.py

```diff
@@ -0,0 +1,21 @@
+
+
+def find_special_state(states):
+    # define the sets of letters in the same line in keyboard (qwerty) in a list
+    list_of_sets = [set("qwertyuiop"), set("asdfghjkl"), set("zxcvbnm")]
+    # find the special state
+    for state in states:
```

Comment:
Separate code to small functions- 
https://softwareengineering.stackexchange.com/questions/308108/when-is-it-appropriate-to-make-a-separate-function-when-there-will-only-ever-be

In general two nested for loops require a better solution or separation